jobsets/solidity.nix

```diff
@@ -0,0 +1,36 @@
+{ pkgs
+, soliditySrc
+, iele
+}:
+
+pkgs.stdenv.mkDerivation {
+  name = "solidity";
+  src = soliditySrc;
+
+  buildInputs = with pkgs; [
+    cmake
+    llvm
+    (boost.override { enableStatic = true; })
+    libtool
+    autoconf
+    automake
+    z3
+    makeWrapper
+  ];
+
+  patches = [ ./solidity.patch ];
+
+  configurePhase = ''
+    export NIX_CFLAGS_COMPILE="-Wno-error=maybe-uninitialized $NIX_CFLAGS_COMPILE"
+    mkdir build
+    cd build
+    cmake ..
```

Comment:
the default `configurePhase` already does most of this when you add cmake to the inputs